pkgs/shells/rc/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, autoreconfHook
+, ncurses #acinclude.m4 wants headers for tgetent().
+, historySupport ? false
+, editSupport ? "readline", readline ? null, libedit ? null
+}:
+
+assert editSupport == "readline" -> readline != null;
```

Comment:
I propose just having a nullable `readline` parameter; setting it to `null` would build an `rc` without line editing and history. If `libedit` or whatever reaches feature parity, we could switch to that in the future.